lib/storage-client.js

```diff
@@ -275,15 +277,15 @@ class ChromedriverStorageClient {
       }
     }
 
-    if (!isNaN(minBrowserVersion)) {
+    if (_.isString(minBrowserVersion) && !Number.isNaN(minBrowserVersion)) {
```

Comment:
if minBrowserVersion is already a string then it cannot be NaN

Reply:
there's no way to express that in TS.